### PR TITLE
Add settings for mumu analysis

### PR DESCRIPTION
**Pull request by ginnocen**

### CODE REVIEW

File: codeHF/PlotEfficiency.C [hunk 1]
```diff
@@ -89,13 +89,13 @@ Int_t PlotEfficiency(TString pathFile = "AnalysisResults.root", TString particle
...
     TH1F* hEff = (TH1F*)hPtRec->Clone("hEff");
-    hEff->Divide(hPtGen);
+    hEff->Divide(hEff, hPtGen, 1., 1., "B");
     hEff->SetTitle(Form("Entries ratio: %g;#it{p}_{T} (GeV/#it{c});efficiency", (double)nRec / (double)nGen));
     yMin = hEff->GetMinimum(0);
     yMax = hEff->GetMaximum();
     SetHistogram(hEff, yMin, yMax, marginRLow, marginRHigh, logScaleR);
     SetPad(padR, logScaleR);
-    hEff->Draw();
+    hEff->Draw("PE");
```
Comment (vkucera) [inline codeHF/PlotEfficiency.C:98]: What do these do?

Comment (ginnocen) [reply codeHF/PlotEfficiency.C:98]: Add binomial error to the efficiencies.

Comment (vkucera) [reply codeHF/PlotEfficiency.C:98]: And the `"PE"` is for a marker with error bars?

Comment (ginnocen) [reply codeHF/PlotEfficiency.C:98]: Yes.

**Comment by vkucera** [comment]
Please fix the error.

File: codeHF/dpl-config_run5_oniaX.json [hunk 2]
```diff
@@ -256,11 +261,13 @@
         "d_pidTPCMaxpT": "2000.",
         "d_pidTOFMinpT": "0.15",
         "d_pidTOFMaxpT": "1.",
+        "d_nSigmaTOFCombined": "0.",
         "d_pidRICHMinpT": "0.5",
         "d_pidRICHMaxpT": "15.",
+        "d_nSigmaRICHTOFCombined": "0.",
```
Comment (vkucera) [inline codeHF/dpl-config_run5_oniaX.json:267]: `nSigmaRICHCombinedTOF`

**Pull request merged**